scripts/aws/setupdb.sh

```diff
@@ -25,7 +26,7 @@ load_stream=false
 load_mapshed=false
 load_water_quality=false
 
-while getopts ":hbsdpmqf:" opt; do
+while getopts ":hbsdpmqfx:" opt; do
```

Comment:
Since `f` also takes a value, doesn't it need to be `f:x:`? That's my takeaway from the docs at least.

>When you want getopts to expect an argument for an option, just place a : (colon) after the proper option flag. If you want -A to expect an argument (i.e. to become -A SOMETHING) just do:
>
>```
>getopts fA:x VARNAME
>```